src/openlcb/TcpDefs.hxx

```diff
@@ -42,8 +42,16 @@ namespace openlcb {
 
 class TcpDefs {
 public:
-    static const char MDNS_SERVICE_NAME_TCP[];
-    static const char MDNS_SERVICE_NAME_GRIDCONNECT_CAN[];
+    // Protocol to be used for mDNS broadcast
+    static const char TcpDefs::MDNS_PROTOCOL_TCP[];
```

Comment:
this should not have TcpDefs:: on it. It does not compile this way.

Reply:
copy and paste error :)